shared-view_components/shared-view_components.gemspec

```diff
@@ -0,0 +1,25 @@
+require_relative "lib/shared/view_components/version"
+
+Gem::Specification.new do |spec|
+  spec.name        = "shared-view_components"
+  spec.version     = Shared::ViewComponents::VERSION
+  spec.authors     = ["GitHub Open Source", "Eric Berry"]
+  spec.email       = ["opensource+view_component@github.com", "eric@berry.sh"]
```

Comment:
Might we change this to be a placeholder email address so that folks don't accidentally have emails coming to us?